src/components/menu-item/menu-item.scss

```diff
@@ -0,0 +1,200 @@
+:host {
+  @apply flex
+  items-center 
+  relative 
+  h-full 
+  box-border;
+}
+
+:host(:not([layout="vertical"])) {
+  flex-shrink: 0;
+  & .container,
+  & .item-content,
+  & a {
+    min-block-size: theme("spacing.12");
+  }
+}
+
+:host([layout="vertical"]) {
+  @apply w-full;
+}
+
+.container,
+.item-content {
+  display: flex;
+  flex-direction: row;
+  block-size: 100%;
+  inline-size: 100%;
+  align-items: stretch;
+}
+
+a {
+  @apply flex
+  h-full
+  items-center
+  relative
+  justify-center
+  cursor-pointer 
+  px-4 
+  outline-none
+  text-0
+  text-color-2 box-border bg-foreground-1;
+  text-decoration: none;
+  border-block-end: theme("spacing[0.5]") solid transparent;
+  padding-block-start: theme("spacing[0.5]");
+  &:hover {
+    @apply bg-foreground-2 text-color-2;
+  }
+  &:focus {
+    @apply bg-foreground-2 text-color-2 focus-inset;
+  }
+  &:active {
+    @apply bg-foreground-3 text-color-1;
+  }
+  & span {
+    display: inline-flex;
+  }
+  &.layout--vertical {
+    @apply py-4 w-full;
+    display: flex;
+    justify-content: flex-start;
+    border-block-end: 0;
+    border-inline-end: theme("spacing.1") solid transparent;
+  }
+}
+
+:host([active]) a {
+  @apply text-color-1;
+  border-color: var(--calcite-ui-brand);
+  .icon {
+    --calcite-ui-icon-color: var(--calcite-ui-brand);
+  }
+}
+
+.icon--start {
+  @apply mr-3;
+}
+
+.icon--end {
+  @apply ml-3;
+}
+
+.icon--dropdown {
+  @apply pl-2 mr-0 ml-auto;
+  --calcite-ui-icon-color: var(--calcite-ui-text-3);
+  position: relative;
+}
+
+:host([layout="vertical"]) .icon--dropdown {
+  inset-inline-start: theme("spacing.1");
+}
+
+.icon--breadcrumb {
+  @apply pl-2 mr-0;
+  --calcite-ui-icon-color: var(--calcite-ui-text-3);
+}
+
+:host([breadcrumb]) a {
+  @apply pr-3;
+}
+
+calcite-action {
+  block-size: auto;
+  position: relative;
+  border-inline-start: 1px solid var(--calcite-ui-foreground-1);
+  &:after {
+    display: block;
+    inline-size: 1px;
+    content: "";
+    inset-block: theme("spacing.3");
+    background-color: var(--calcite-ui-border-3);
+    position: absolute;
+    inset-inline-start: -1px;
+    z-index: 9;
+  }
+  &:hover:after {
+    block-size: 100%;
+    inset-block: 0;
+  }
+}
+
+a:focus ~ calcite-action,
+a:hover ~ calcite-action {
+  @apply text-color-1;
+  border-inline-start: 1px solid var(--calcite-ui-border-3);
+}
+
+.container:hover .dropdown-action {
+  @apply bg-foreground-2;
+}
+
+.dropdown-menu-items {
+  border: 1px solid var(--calcite-ui-border-3);
+  position: absolute;
+  background: var(--calcite-ui-foreground-1);
+  z-index: 9;
+  block-size: auto;
+  flex-direction: column;
+  min-inline-size: 100%;
+  display: none;
+  transition: all var(--calcite-internal-animation-timing-slow) ease-in-out;
+  overflow: visible;
+  inset-block-start: 100%;
+
+  &.open {
+    display: block;
+  }
+  &.nested {
+    z-index: 9;
```

Comment:
Why a zindex of 9?

Reply:
The z-index were added during initial prototypes, will be removed. 

cc: @macandcheese